### PR TITLE
system_platform and system_advisories partitioning

### PR DESCRIPTION
**Pull request by MichaelMraka**

### CODE REVIEW

File: database_admin/schema/create_schema.sql [hunk 1]
```diff
@@ -667,24 +665,35 @@ ON CONFLICT DO NOTHING;
...
+) PARTITION BY LIST (when_patched IS NOT NULL);
+
+CREATE TABLE system_advisories_unpatched PARTITION OF system_advisories
+  FOR VALUES IN (false) WITH (fillfactor = '70', autovacuum_vacuum_scale_factor = '0.05')
   TABLESPACE pg_default;
 
+CREATE TABLE system_advisories_patched PARTITION OF system_advisories
+  FOR VALUES IN (true) PARTITION BY HASH (rh_account_id);
+
+SELECT create_table_partitions('system_advisories_patched', 8,
```
Comment (semtexzv) [inline database_admin/schema/create_schema.sql:693]: We need partitions for the unpatched variant.These are shown to user

**Comment by MichaelMraka** [comment]
rebased to v1.4.1

**Review by Josca — APPROVED** [review]
Greate job @mim, thanks.

**Comment by MichaelMraka** [comment]
Oops, I've just found out we have to move sequence on new system_platform to where it was on old table before migration.

**Comment by Josca** [comment]
@MichaelMraka oh, it's seen that was something what was not covered by tests, hmmm :thinking:

**Comment by MichaelMraka** [comment]
Just comparing sequences is not enough.
This could be found only if we had some pre-filled data in schema comparison/migration test.

**Pull request merged**